_includes/manuals/1.19/1.2_release_notes.md

```diff
@@ -221,70 +175,92 @@ Previously, the cloud-init integration for oVirt supported only a sub-set of clo
 * Improve assert in host test ([#23834](https://projects.theforeman.org/issues/23834))
 * Improve wait_for_ajax ([#23756](https://projects.theforeman.org/issues/23756))
 * sd_notify_test may fail with "too long unix socket path (109 bytes given but 108 bytes max)" error ([#23666](https://projects.theforeman.org/issues/23666))
 * Stop doing bundle install in Travis builds ([#23643](https://projects.theforeman.org/issues/23643))
+* Port robottelo tests for classparameters ([#23609](https://projects.theforeman.org/issues/23609))
+* Port robottelo tests for smart_proxy ([#23605](https://projects.theforeman.org/issues/23605))
+* Port robottelo tests for template combination ([#23597](https://projects.theforeman.org/issues/23597))
+* Port robottelo tests for template ([#23586](https://projects.theforeman.org/issues/23586))
+* Port robottelo tests for smart variables ([#23565](https://projects.theforeman.org/issues/23565))
 * Use puma for Capybara tests ([#23518](https://projects.theforeman.org/issues/23518))
+* Port robottelo tests for filters ([#23499](https://projects.theforeman.org/issues/23499))
+* Port robottelo tests for interfaces ([#23485](https://projects.theforeman.org/issues/23485))
 * update capybara to 3.x ([#23474](https://projects.theforeman.org/issues/23474))
 * update show_me_the_cookies to 4.x ([#23469](https://projects.theforeman.org/issues/23469))
+* Port robottelo tests for hostgroup ([#23419](https://projects.theforeman.org/issues/23419))
 * Tests randomly fail with unexpected invocation: statuses() ([#23379](https://projects.theforeman.org/issues/23379))
+* Port robottelo tests for Role ([#23350](https://projects.theforeman.org/issues/23350))
 * Use memory cache store instead file store in tests ([#23346](https://projects.theforeman.org/issues/23346))
+* Port robottelo tests for hosts ([#23339](https://projects.theforeman.org/issues/23339))
+* Robottelo tests reporting generation ([#23295](https://projects.theforeman.org/issues/23295))
 * speed up seeds test ([#23255](https://projects.theforeman.org/issues/23255))
+* Utilize webmock to prevent network connection during tests ([#23254](https://projects.theforeman.org/issues/23254))
 
 #### Trends
 * trends welcome page text  should not be limited to puppet facts ([#23467](https://projects.theforeman.org/issues/23467))
+* API to configure trends ([#18429](https://projects.theforeman.org/issues/18429))
 
 #### Unattended installations
 * Issue warning when multiple hosts with same MAC are found ([#23177](https://projects.theforeman.org/issues/23177))
-
-#### Uncategorized
-* Remove storeconfigs support ([#23646](https://projects.theforeman.org/issues/23646))
-* Optionally load facter ([#23635](https://projects.theforeman.org/issues/23635))
-* Use a group for bunder.d/assets.rb ([#23579](https://projects.theforeman.org/issues/23579))
+* Templates API to to mark a build as failed ([#21007](https://projects.theforeman.org/issues/21007))
```

Comment:
I've updated the issue to remove the duplicate `to`

Reply:
Thanks, i'll merge this as is for now, we'll pull in the corrected version for the next release.